daq/host.py

```diff
@@ -194,6 +212,10 @@ def _get_static_ip(self):
     def _get_dhcp_mode(self):
         return self._loaded_config['modules'].get('ipaddr', {}).get('dhcp_mode', 'normal')
 
+    def _get_dhcp_tests(self):
+        tests = self._loaded_config.get('dhcp_tests', {}).keys()
```

Comment:
A related fundamental question I have is if "ipaddr" and "dhcp_tests" are conceptually related. They seem to fall into the same conceptual bucket (things that relate to the ipaddress of the device) so wondering if there's a cleaner consolidation that streamlines things?